studio.html

```diff
@@ -6,13 +6,9 @@
         <meta name="robots" content="noindex, nofollow" />
         <meta name="nofollow-links" content="on" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <meta name="aem-base-url"/>
-        <script>
-            let baseUrl = window.location.href.includes('localhost') ? 'http://localhost:8080' : 'https://mas-proxy-odin.corp.adobe.com';
```

Comment:
does it work on localhost as well without proxy? 
when we switch prod we'll still need this i think